examples/tsunami/chile2010/Makefile

```diff
@@ -23,7 +23,8 @@ RESTART = False
 # Environment variable FC should be set to fortran compiler, e.g. gfortran
 
 # Compiler flags can be specified here or set as an environment variable
-FFLAGS ?=
+FFLAGS = -O3 -funroll-loops -finline-functions -fopenmp -cpp -DNETCDF -I$(NETCDF4_DIR)/include
+LFLAGS = -fopenmp -lnetcdff
```

Comment:
Presumably we should not have this netcdf dependency in the standard chile2010 example?